CircuitPython_Pyloton/code.py

```diff
@@ -0,0 +1,63 @@
+from time import time
+import adafruit_ble
+import board
+import pyloton
+
+# PyLint can't find BLERadio for some reason so special case it here.
+ble = adafruit_ble.BLERadio()    # pylint: disable=no-member
+
+display = board.DISPLAY
+
+CONNECTION_TIMEOUT = 45
+
+HEART = True
+SPEED = True
+CADENCE = True
+AMS = True
+DEBUG = False
+
+# 84.229 is wheel circumference (700x23 in my case)
+pyloton = pyloton.Pyloton(ble, display, 84.229, HEART, SPEED, CADENCE, AMS, DEBUG)
+
+pyloton.show_splash()
+
+ams = pyloton.ams_connect()
+
+
+start = time()
+hr_connection = None
+speed_cadence_connections = []
+radio = None
```

Comment:
is this variable used anywhere?

Reply:
Yeah, it's used in line 38 (will be 35 after next push) so there isn't a NameError.

Reply:
i was referring to 'radio = None'

Reply:
Ah, ok